.gitignore

```diff
@@ -1,5 +1,5 @@
 .idea/
 out/
 .gradle/
-build/
+build/git
```

Comment:
You added in a lot of files here that you shouldn't have because you changed this line.